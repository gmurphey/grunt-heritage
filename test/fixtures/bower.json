{
  "name": "old plugin name",
  "version": "0.0.0"
}
