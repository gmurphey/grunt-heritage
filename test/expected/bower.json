{
  "name": "fancy plugin",
  "version": "0.1.0",
  "description": "a description"
}

